docs/reference/metadata.md

```diff
@@ -383,11 +383,6 @@ Example:
 
 ### Subjects (0-n)
```

Comment:
off-topic: I honestly wonder why are they displayed like this 🤔 with all the red background...

Reply:
Me too. It's not new, but I figured it out and fixed it just now.